config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,20 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I generated a new Rails 5 app, and noticed that this file is actually generated. If one of these options isn't set manually in this file, it's `nil`, not the default value. This doesn't mean the default behavior isn't still used, but it felt safest to leave this file.